api/_lighthouse_ide_helper.php

```diff
@@ -155,12 +155,3 @@ public function assertGraphQLValidationPasses(): self
         }
     }
 }
-
```

Comment:
> This file should be ignored by git as it can be autogenerated.

Is this comment still relevant or should this file be added to `.gitignore`?

Reply:
I can't be certain but I think it should be safe to add it to `.gitignore`

Reply:
Are we checking this in to make some CI easier?

Reply:
That is a very good question. I think you may be correct but I am not sure who would be able to verify this 🤷‍♀️